database/config.json

```diff
@@ -1,14 +1,14 @@
 {
     "development": {
-        "username": "student",
-        "password": null,
+        "username": "root",
+        "password": "",
         "database": "trektracker",
         "host": "localhost",
         "dialect": "mysql"
     },
     "test": {
         "username": "root",
-        "password": null,
+        "password": "",
```

Comment:
Group note: acknowledgement we will update test object to have "root" as username and password as null acc: to original master repo.